models/mrs_robots_description/sdf/README.md

```diff
@@ -0,0 +1,48 @@
+## Camera frame and Topic naming
+### RGB Camera
+Due to different frame conventions, an additional optical frame is required. You may change its name, since the TF publisher will read it from the SDF file. For example, I recommend checking the [bluefox_camera_macro](./components/camera/bluefox.sdf.jinja).
+```
+{%- set camera_gz_frame_id = spawner_args['name'] + '/' + camera_name + '/color_optical' -%}
+```
+
+The resulting camera-related topics will look like as follows:
```

Comment:
This is relevant to the gazebo topic; perhaps we could also discuss how it is implemented on the ROS side.